modules/core/src/v2/coins/cgld.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @prettier
+ */
+import { BaseCoin } from '../baseCoin';
+import { BitGo } from '../../bitgo';
+import { BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+import { AbstractEthLikeCoin } from './abstractEthLikeCoin';
+
+export class Cgld extends AbstractEthLikeCoin {
+  protected constructor(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo, staticsCoin);
+  }
+
+  static createInstance(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new AbstractEthLikeCoin(bitgo, staticsCoin);
```

Comment:
Should return a `Cgld` instance, not an instance of the abstract base. (Same for other subclasses)